pipeline-modules/deployment-service/azure/artifact-config.yaml

```diff
@@ -30,6 +30,11 @@ inputs:
       title: Artifact type
       description: Type of artifact
       type: string
+    pool_name:
```

Comment:
you shouldn't need to put it in this module - it is internal

Reply:
removed